cloudsim-plus-testbeds/src/main/java/hostFaultInjection/HostFaultInjectionExperiment.java

```diff
@@ -324,15 +324,19 @@ public double getRatioVmsPerHost() {
         return vmsize / hostsize;
     }
 
+    /**
+     * Computes the percentage of customers for whom the availability stated
+     * in the SLA was met (in scale from 0 to 1, where 1 is 100%).
+     * @return
+     */
     public double getPercentageOfAvailabilityThatMeetingTheSla() {
         double total = 0;
         double totalOfAvailabilitySatisfied = getBrokerList()
             .stream()
             .map(b -> faultInjection.availability(b) * 100)
```

Comment:
Your method keep multiplying percentage values by 100. 
To avoid bugs and confusion (because the entire framework uses percentages as values from 0 to 1), you have to multiple the value by 100 only, and just only when you need to print them.